pkg/doc/verifiable/credential.go

```diff
@@ -188,7 +189,41 @@ const jsonSchema2018Type = "JsonSchemaValidator2018"
 const vpType = "VerifiablePresentation"
 
 //nolint:gochecknoglobals
-var defaultSchemaLoader = gojsonschema.NewStringLoader(defaultSchema)
+var (
+	defaultSchemaLoader = gojsonschema.NewStringLoader(defaultSchema)
+
+	// jsonSchemaCache keeps cache of JSON schema by URL
+	jsonSchemaCache = schemaCache{schemaBytes: make(map[string][]byte)}
```

Comment:
@fqutishat  Should we use different implementation of cache?

Reply:
@kdimak  use https://github.com/VictoriaMetrics/fastcache

Reply:
@troyronda

Reply:
@sandrask @fqutishat @troyronda I checked `aries` and `fabric`, did not find any usage of an external cache library like https://github.com/VictoriaMetrics/fastcache.
And we have `memStore` struct at pkg/storage/mem/mem_store.go which I looked at and reworked for schemas cache here.

I don't know https://github.com/VictoriaMetrics/fastcache library. If you recommend it, why not to make use of it?

Reply:
@kdimak we don't use any caching tools yet your PR is the first one

Reply:
@kdimak  https://github.com/hyperledger/fabric/commit/3a8dae88b9b676856da71d2a5af20ce97212c553